CONTRIBUTING.md

```diff
@@ -38,10 +38,10 @@ Checkout a new branch, make modifications, build locally, and push the branch to
 to open a new PR:
 
 ```shell
-$ git checkout -b feature
+$ git checkout -b myfeature
 # edit
-$ make
-$ make fmt
-$ git commit
-$ git push fork feature
-```
\ No newline at end of file
+# verify that all applications are running well and emitting expected telemetry
```

Comment:
nit, this comment could suggests the following does this, so referencing an actionable bit may be helpful too.